extensions/console-cli.crd.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: consoleclitools.console.openshift.io
+  annotations:
+    displayName: ConsoleCLITool
+    description: Extension for configuring openshift web console command line interface (CLI) tools.
+spec:
+  group: console.openshift.io
+  version: v1
+  versions:
+  - name: v1
+    served: true
+    storage: true
+  scope: Cluster
+  names:
+    plural: consoleclitools
+    singular: consoleclitool
+    kind: ConsoleCLITool
+    listKind: ConsoleCLIToolList
+  additionalPrinterColumns:
+  - name: Text
+    type: string
+    JSONPath: .spec.text
+  - name: Age
+    type: date
+    JSONPath: .metadata.creationTimestamp
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          type: object
+          description: Represents console cli tool spec
+          required:
+          - name
+          - description
+          - link
+          properties:
+            name:
+              type: string
+              description: Name of the cli tool
+            description:
+              type: string
+              description: Description of the cli tool
+            link:
```

Comment:
I'm trying to decide if we need multiple links for different platforms. This is probably fine, but asking the question because it's difficult to change later.

Reply:
Better to have an array of one than to not have the ability to change?

Reply:
Prefer continuing discussion https://github.com/openshift/console-operator/pull/229/files#r286672961 since the `manifest` will be removed from this PR.

Reply:
If we make it an array, we need to support showing all values if set. Otherwise it just seems like a bug :)

Reply:
@spadgett, I defer to you on link vs. links.  What'll it be?

Reply:
Updated to links, an array.